repo/packages/A/artifactory-secondary/0/marathon.json.mustache

```diff
@@ -0,0 +1,86 @@
+  {{#artifactory-secondary.high-availability.enabled}}
+  "instances": {{artifactory-secondary.high-availability.nodes}},
+  {{/artifactory-secondary.high-availability.enabled}}
+   "container": {
+       "type": "DOCKER",
+       "docker": {
+           {{^artifactory-secondary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.e183c36bdb4b}}",
+           {{/artifactory-secondary.high-availability.enabled}}
+           {{#artifactory-secondary.high-availability.enabled}}
+           "image": "{{resource.assets.container.docker.a40023d9e984}}",
+           {{/artifactory-secondary.high-availability.enabled}}
+           "network": "HOST",
+            "forcePullImage":true
+       },
+       "volumes": [
+          {{^artifactory-secondary.high-availability.enabled}}
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/data",
+             "hostPath": "{{artifactory-secondary.artifactory-home}}/{{artifactory-secondary.framework-name}}/data",
+             "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/logs",
+             "hostPath": "{{artifactory-secondary.artifactory-home}}/{{artifactory-secondary.framework-name}}/logs",
+             "mode": "RW"
+          },
+          {
+             "containerPath": "/var/opt/jfrog/artifactory/backup",
+             "hostPath": "{{artifactory-secondary.artifactory-home}}/{{artifactory-secondary.framework-name}}/backup",
+             "mode": "RW"
+          }
+          {{/artifactory-secondary.high-availability.enabled}}
+          {{#artifactory-secondary.high-availability.enabled}}
+          {
+             "containerPath": "/var/opt/jfrog/cluster",
+             "hostPath": "{{artifactory-secondary.high-availability.cluster-home}}/cluster-home",
+             "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/logs",
+          "hostPath": "{{artifactory-secondary.high-availability.cluster-home}}/{{artifactory-secondary.framework-name}}/logs",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "//var/opt/jfrog/artifactory/data",
+          "hostPath": "{{artifactory-secondary.high-availability.cluster-home}}/{{artifactory-secondary.framework-name}}/data",
+          "mode": "RW"
+          },
+          {
+          "containerPath": "/var/opt/jfrog/artifactory/backup",
+          "hostPath": "{{artifactory-secondary.high-availability.cluster-home}}/{{artifactory-secondary.framework-name}}/backup",
+          "mode": "RW"
+          }
+          {{/artifactory-secondary.high-availability.enabled}}
+       ]
+   },
+  "labels": {
+      "HAPROXY_GROUP":"external"
+   },
+   "env": {
+    "PRIMARY_BASE_URL": "{{artifactory-secondary.artifactory-primary-url}}"
+   }
+,"healthChecks": [
+{
+"protocol": "COMMAND",
+"command": {
+"value": "curl http://$HOST:8081/artifactory/api/system/ping"
```

Comment:
Using $PORT0 here instead of 8081 will allow you to migrate to dynamic ports more easily in the future.